lib/sasl/test/systools_SUITE.erl

```diff
@@ -945,12 +945,48 @@ system_files_tar(Config) ->
 
     ok.
 
+
 system_files_tar(cleanup,Config) ->
     Dir = ?privdir,
     file:delete(filename:join(Dir,"sys.config")),
     file:delete(filename:join(Dir,"relup")),
     ok.
 
+%% make_tar: Check that sys.config.src and not sys.config is included
+system_src_file_tar(Config) ->
+    {ok, OldDir} = file:get_cwd(),
+
+    {LatestDir, LatestName} = create_script(latest,Config),
+
+    DataDir = filename:absname(?copydir),
+    LibDir = fname([DataDir, d_normal, lib]),
+    P = [fname([LibDir, 'db-2.1', ebin]),
+         fname([LibDir, 'fe-3.1', ebin])],
+
+    ok = file:set_cwd(LatestDir),
+
+    %% Add dummy relup and sys.config
+    ok = file:write_file("sys.config.src","[${SOMETHING}].\n"),
+    ok = file:write_file("sys.config","[].\n"),
+
+    {ok, _, _} = systools:make_script(LatestName, [silent, {path, P}]),
+    ok = systools:make_tar(LatestName, [{path, P}]),
+    ok = check_tar(fname(["releases","LATEST","sys.config.src"]), LatestName),
+    {error, _} = check_tar(fname(["releases","LATEST","sys.config"]), LatestName),
+    {ok, _, _} = systools:make_tar(LatestName, [{path, P}, silent]),
+    ok = check_tar(fname(["releases","LATEST","sys.config.src"]), LatestName),
+    {error, _} = check_tar(fname(["releases","LATEST","sys.config"]), LatestName),
+
+    ok = file:set_cwd(OldDir),
+
+    ok.
+
+system_src_files_tar(cleanup,Config) ->
```

Comment:
This function must have the same name as the test case function - i.e. system_src_file_tar (no s after file). It is called [here](https://github.com/tsloughter/otp/blob/1559adbef20f69c78e8f2b4f9ba50c5c87069dfa/lib/sasl/test/systools_SUITE.erl#L138).